src/grdcut.c

```diff
@@ -31,7 +31,7 @@
 #define THIS_MODULE_MODERN_NAME	"grdcut"
 #define THIS_MODULE_LIB		"core"
 #define THIS_MODULE_PURPOSE	"Extract subregion from a grid"
-#define THIS_MODULE_KEYS	"<G{,FD(=,GG}"
+#define THIS_MODULE_KEYS	"<G{,FD(=,>D),GG}"
```

Comment:
```suggestion
#define THIS_MODULE_KEYS	"<G{,FD(=,>DD,GG}"
```
Still working on understanding these macros, but I think `Z` in `XYZ` should be `D` to shift primary output if -D is given rather than just adding a secondary output option.